### PR TITLE
fix: rewrite statusText check to status value

### LINKED ISSUE #24: Clients page: Failed to fetch clients. undefined
**Description**
If I open in administration "Clients" overview page it fails on:
`Failed to fetch clients. undefined`

**Problem**
I have find out that on request works correctly:
```
Request URL:  /api/seacat_auth/client?p=1&i=20&f=
Request Method: GET
Status Code: 200
``` 

And the difference against localhost is that `status code` didn't return "200 OK" but only "200".
In a code, I have find out that there is the "statusText" check which is failing.

> See [HTTP response status codes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status) for a list of codes and their associated status messages. Note that HTTP/2 [does not support](https://fetch.spec.whatwg.org/#concept-response-status-message) status messages.

see https://developer.mozilla.org/en-US/docs/Web/API/Response/statusText

_Solution:_
> Drop response.statusText and use response.status as solution.

see https://github.com/axios/axios/issues/1501

**DoD**
 - replace statusText check with status

### PR DESCRIPTION
**Pull request by elpablos**
close #24

### CODE REVIEW

File: src/modules/auth/clients/ClientCreateContainer.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ const ClientCreateContainer = (props) => {
 	const retrieveClientFeatures = async () => {
 		try {
 			let response = await SeaCatAuthAPI.get('/client/features');
-			if (response.statusText != 'OK') {
+			if (response.status !== 200) {
```
Comment (Pe5h4) [inline src/modules/auth/clients/ClientCreateContainer.js]: @elpablos As for `get` calls, the condition should go away completely :) Meaning for this particular case:

```
let response = await SeaCatAuthAPI.get('/client/features');
setMetaData(response.data["metadata_schema"]);
setTemplate(response.data["templates"]);
```

File: src/modules/auth/clients/ClientCreateContainer.js [hunk 2]
```diff
@@ -142,7 +142,7 @@ const ClientCreateContainer = (props) => {
 		let body = refactorSubmitData(values, "create");
 		try {
 			let response = await SeaCatAuthAPI.post(`/client`, body);
-			if (response.statusText != 'OK') {
+			if (response.status !== 200) {
```
Comment (Pe5h4) [inline src/modules/auth/clients/ClientCreateContainer.js]: @elpablos As for `post` and `put` calls, the condition should be

```
if (response.data.result !== "OK") {
	...
}
```

Comment (Pe5h4) [reply src/modules/auth/clients/ClientCreateContainer.js]: Please validate with @byewokko if those affected `post` and `put` endpoints returns `result`

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
Thank you for your findings, I have proposed expected refactorisation in the comments

File: src/modules/auth/clients/ClientCreateContainer.js (hunk 2 above)
Comment (byewokko) [reply src/modules/auth/clients/ClientCreateContainer.js]: - `POST /client/<client_id>` does **not** return `result` field
- `PUT /client/<client_id>` does return `result` field
- `DELETE /client/<client_id>` does return `result` field

Comment (Pe5h4) [reply src/modules/auth/clients/ClientCreateContainer.js]: So, if there is no `result` for `POST` in client , then the condition is redundant and can be removed as per example in `GET`

Comment (elpablos) [reply src/modules/auth/clients/ClientCreateContainer.js]: Sorry for delay, alots of work out of seacat projects. Thanks for research.
Its already test it and check it againts seacat API impl.

Comment (Pe5h4) [reply src/modules/auth/clients/ClientCreateContainer.js]: @elpablos No worries, thank you for your effort and for testing!

**Review by Pe5h4 — APPROVED** [review]
@elpablos I added changelog, all discussions are resolved, it can be merged :)

**Pull request merged**